thermo_scenes/scripts/preprocess_thermal.py

```diff
@@ -20,10 +20,16 @@ class Paths:
 def main() -> None:
     paths = tyro.cli(Paths)
 
-    _ = CustomFlir(
+    custom_flir = CustomFlir(
         path_to_msx_images=paths.msx_images,
         path_to_output_folder=paths.output_folder,
     )
+    if custom_flir.horizontal:
```

Comment:
Let's re-write this so it's not an if/else here but just a if is needed.

Also why are those number hardcoded? Not all cameras have the same height and width so it could we extract from the metadata instead?